variables.tf

```diff
@@ -94,16 +99,10 @@ variable "access_group_name" {
   default     = "client-to-site-vpn-access-group"
 }
 
-variable "create_s2s_auth_policy" {
+variable "skip_secrets_manager_iam_auth_policy" {
   type        = bool
-  description = "Create IAM Service to Service Authorization to allow communication between all VPN Servers (scoped to the given resource group) and the given Secrets Manager instance. Currently not possible to scope the policy to the exact VPN server ID since the policy is needed before the instance exists as it uses the cert stored in secrets manager during the provisioning process."
-  default     = true
-}
-
-variable "secrets_manager_id" {
-  type        = string
-  description = "ID of the Secrets Manager that contains the certificate to use for the VPN, only required when create_s2s_auth_policy is true."
-  default     = null
+  description = "Whether to create an IAM authorization policy that permits communication between all VPN Servers (scoped to the given resource group) and the given Secrets Manager instance. Currently not possible to scope the policy to the exact VPN server ID since the policy is needed before the instance exists as it uses the cert stored in secrets manager during the provisioning process."
```

Comment:
should probably give more info on the exact role it grants instead of just saying "permits communication"

Reply:
updated